plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-inventory.php

```diff
@@ -10,7 +10,11 @@
 }
 ?>
 <div id="inventory_product_data" class="panel woocommerce_options_panel hidden">
-
+	<div class="inline notice woocommerce-message show_if_variable">
+		<p>
+			<?php echo esc_html_e( 'Settings below apply to all variations without manual stock management enabled. ', 'woocommerce' ); ?> <a target="_blank" href="https://woocommerce.com/document/variable-product/"><?php esc_html_e( 'Learn more', 'woocommerce' ); ?></a>
```

Comment:
There is an errant space at the end of the translated message. `enabled. ` That space is not needed.